src/helm/README.md

```diff
@@ -0,0 +1,37 @@
+# Ralph Helm Chart
+
+This is the helm chart used to deploy ralph application.
+
+All default values are in `values.yaml`
+## Review manisfest
+To generate and review your manifest, under `./src/helm`
+```
+helm template .
```

Comment:
We usually add the prompt so that users are aware that these are shell commands:

```suggestion
$ helm template .
```